kubetest/local.go

```diff
@@ -27,57 +26,13 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
-	"sort"
 	"time"
-
-	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/client"
 )
 
 var (
 	localUpTimeout = flag.Duration("local-up-timeout", 2*time.Minute, "(local only) Time limit between 'local-up-cluster.sh' and a response from the Kubernetes API.")
 )
 
-func removeAllContainers(cli *client.Client) {
-	// list all containers
-	listOptions := types.ContainerListOptions{
-		Quiet: true,
-		All:   true,
-	}
-	containers, err := cli.ContainerList(context.Background(), listOptions)
-	if err != nil {
-		log.Printf("Failed to list containers: %v\n", err)
-		return
-	}
-
-	// reverse sort by Creation time so we delete newest containers first
-	sort.Slice(containers, func(i, j int) bool {
-		return containers[i].Created > containers[j].Created
-	})
-
-	// stop then remove (which implicitly kills) each container
-	duration := time.Second * 1
-	removeOptions := types.ContainerRemoveOptions{
-		RemoveVolumes: true,
```

Comment:
would we also need `-v` in the `docker rm` replacement to maintain the same behavior ?

Reply:
Good catch!